src/components/WalletConnection/WalletSelector.tsx

```diff
@@ -150,6 +161,57 @@ export const WalletSelector = () => {
       />
       <WalletRow key="torus_wallet" walletName="Torus" walletType={WalletType.TORUS} />
       <WalletRow key="frame_wallet" walletName="Frame" walletType={WalletType.FRAME} />
+      <Box sx={{ display: 'flex', alignItems: 'center', mb: 1, padding: '10px 0' }}>
+        <Typography variant="subheader1" color="text.secondary">
+          <Trans>Enter an address to track in watch-only mode</Trans>
+        </Typography>
+        <WatchOnlyModeTooltip />
+      </Box>
+      <Box
+        sx={(theme) => ({
+          p: '8px 12px',
+          border: `1px solid ${theme.palette.divider}`,
+          borderRadius: '6px',
+          mb: 1,
+        })}
+      >
+        <InputBase
```

Comment:
One thing I noticed on mobile, iOS is zooming in on this input automatically. Once you enter the address and start watching, then you have to zoom back out. We could conditionally set the font size of the input to be `16px` if we're on small screens, then it won't zoom on focus. I just did something similar for the market search input.